saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -72,9 +72,9 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
         user = cast(User, user)
         order = cls.get_instance(info, **data)
         cls.check_channel_permissions(info, [order.channel_id])
-        order.status = OrderStatus.UNFULFILLED
+        order = update_order_status(order)
         update_order_display_gross_prices(order)
```

Comment:
Hhm, it sounds strange, that we can change the behavior of the `order.display_gross_prices` when calling orderConfirm 🤔 
`update_order_display_gross_prices` is called when we convert checkout to order, or when calling draft create/draft complete. The unconfirmed order already has this value, so not sure why we want to change it here.

I know that this is not the scope of this PR, but I pointed it out, as in case of removing this, we will be able to also remove additional `.save()` call.

Reply:
It's in case order billing or shipping address has changed. So seems like the `update_order_display_gross_prices` should be moved to `OrderUpdate` mutation instead of updating it here. I will create a ticket for that to handle it separately.